src/AccessibilityInsights.Extensions/Interfaces/IssueReporting/IIssueReporting.cs

```diff
@@ -46,9 +46,10 @@ public interface IIssueReporting
         Task RestoreConfigurationAsync(string serializedConfig);
 
         /// <summary>
-        /// Control to let user configure/login to issue reporting service
+        /// Control to let user configure/login to issue reporting service. 
+        /// UpdateSaveButton action needs to be called when the extension is ready to save.
         /// </summary>
-        IssueConfigurationControl ConfigurationControl { get; }
+        IssueConfigurationControl RetrieveConfigurationControl(Action UpdateSaveButton);
```

Comment:
I assume that CanSave is the flag that the save button would use for enabling/disabling, assuming also that the SaveButton is out of th ConfigurationControl, why the control need to know about this action?

Reply:
We haven't had much luck with the flag so thought of doing it this way.
We currently do it for the connection control as well where we pass an action that the configuration control can invoke to update the save button state.